include/openthread/platform/ble.h

```diff
@@ -152,6 +169,26 @@ otError otPlatBleDisable(otInstance *aInstance);
  * @section Bluetooth Low Energy GAP.
  ***************************************************************************/
 
+/**
+ * Starts BLE Advertising procedure.
+ *
+ * The BLE device shall use undirected advertising with no filter applied.
+ * A single BLE Advertising packet must be sent on all advertising
+ * channels (37, 38 and 39).
+ *
+ * @note This function shall be used only for BLE Peripheral role.
+ *
+ * @param[in] aInstance          The OpenThread instance structure.
+ * @param[in] aAdvertisementData The formatted TCAT advertisement frame.
+ * @param[in] aAdvertisementLen  The TCAT advertisement frame length.
+ *
+ * @retval OT_ERROR_NONE           Advertising procedure has been started.
+ * @retval OT_ERROR_INVALID_STATE  BLE Device is in invalid state.
+ * @retval OT_ERROR_INVALID_ARGS   Invalid interval value has been supplied.
+ *
+ */
+otError otPlatBleGapAdvSetData(otInstance *aInstance, uint8_t *aAdvertisementData, uint16_t aAdvertisementLen);
```

Comment:
some issues here in the parameters, and its relation to the comments above, and relation to the other function `otPlatBleGapAdvStart` below here which looks very similar.
* this function should not start BLE Advertising -> that's for  otPlatBleGapAdvStart ?
* "Invalid interval value has been supplied." -> the aInterval parameter is in the other function, not here

Reply:
Fixed.